pkg/cmd/provider/install.go

```diff
@@ -212,8 +215,22 @@ func InstallProvider(apiClient *apiclient.APIClient, providerToInstall provider_
 			return apiclient_util.HandleErrorResponse(res, err)
 		}
 
+		if userAborted { // If ctrl+c is pressed
+			if _, err := goos.Stat(downloadPath); err == nil {
```

Comment:
We can't rely on this because the user might be connected to a remote server (not running on their machine). All file operations must be done on the server side.